Backpack/Alert/Classes/BPKAlertView.m

```diff
@@ -0,0 +1,235 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKAlertView.h"
+#import <Backpack/Label.h>
+#import <Backpack/Spacing.h>
+#import <Backpack/Shadow.h>
+
+
+@interface BPKAlertView()
+
+@end
+
+@implementation BPKAlertView
+{
+    UIView *_circularView;
+    UIView *_iconContainerView;
+    UIImageView *_iconImageView;
+    
+    UIView *_contentHolderView;
+    BPKLabel *_titleLabel;
+    BPKLabel *_descriptionLabel;
+    NSArray<BPKButton *> *_buttons;
+    UIStackView *_buttonStackView;
+    
+    NSDictionary<BPKAlertButtonConfiguration *, BPKButton *> *_buttonConfigurationMap;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    self = [super initWithFrame:frame];
+    if (self) {
+        [self setupViews];
+        [self addViews];
+        [self setupConstraints];
+        
+        
+        UITapGestureRecognizer *tap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(tapView:)];
+        tap.delegate = self;
+        [self setUserInteractionEnabled:YES];
+        [self addGestureRecognizer:tap];
+    }
+    return self;
+}
+
+- (void)setupViews {
+    _contentHolderView = [UIView new];
+    _contentHolderView.backgroundColor = BPKColor.white;
+    _contentHolderView.layer.cornerRadius = 4;
+    
+    _circularView = [[UIView alloc] initWithFrame:CGRectZero];
+    _circularView.layer.cornerRadius = 35;
```

Comment:
Why a hardcoded 35? Shouldn't this be width or height divided by 2?

Reply:
this should be the half of the _circularView frame width. fixed.